students/Josh_HOff/lesson08/circle_class.py

```diff
@@ -0,0 +1,134 @@
+import sys
+import copy
+import math
+import pathlib
+from functools import total_ordering
+
+circles = []
+@total_ordering
+class Circle(object):
+
+    def __init__(self, value):
+        global circles
+        global templist
+        self._radius = value
+        self._diameter = value * 2
+        self._area = (math.pi) * (value ** 2)
+    
+    @staticmethod
+    def from_diameter(value):
+        return Circle(value/2)
```

Comment:
this should be a classmethod, not staticmethod with input arg being `cls`

Reply:
Okay - changed this to class method